src/config/index.ts

```diff
@@ -27,6 +27,7 @@ const config = {
   ensRPC: String(process.env.NEXT_PUBLIC_ENS_RPC),
   publicDomain: String(process.env.NEXT_PUBLIC_DOMAIN),
   treasuryAddress: process.env.NEXT_PUBLIC_TREASURY_ADDRESS,
+  fraxtalTreasuryAddress: process.env.NEXT_PUBLIC_FRAX_TREASURY_ADDRESS,
```

Comment:
Not in this PR, but it'll be nice to setup envs from vercel as well as using env.mjs using ``@t3-oss/env-core``

Reply:
Alright